<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AWS-PVDI-25" failures="1" tests="1" name="Default test" time="41.926" errors="0" timestamp="2023-11-03T09:49:34 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="27.427">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at stepdefinitions.Murabha_AppDataEntery_CustomerFinancials.user_check_employer_phone_extension_field(Murabha_AppDataEntery_CustomerFinancials.java:138)
at âœ½.User_607 Check Employer Phone Extension Field(file:///C:/Users/inindc00607/git/ULS_Ijarah_1/Gowtham_ULS_New/src/test/java/features/Murabha_AppDataEntery_CustomerFinancials.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
